src/content/examples/en/02_Animation_And_Variables/00_Drawing_Lines/description.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 featuredImage: "../../../images/featured/02_Animation_And_Variables-00_Drawing_Lines-thumbnail.png"
+featuredImageAlt: A squiggly rainbow line on a black background

```

Comment:
Thanks @calliecramer for your work on this—the alt text looks fantastic. To keep everything consistent, we'd like all alt text to be formatted as complete sentences with proper punctuation. I noticed that some alt text includes a period while others don’t. Could you please add periods to those that don’t have them yet? Thanks!

Reply:
Thanks for catching that! Punctuation has been added.